genDockerfile.sh

```diff
@@ -0,0 +1,25 @@
+export CI_CONTAINER_VERSION=v4.6.0
+export CI_X_VERSION=4
+export CI_Y_VERSION=6
+export CI_Z_VERSION=0
+
+export CI_CLUSTER_LOGGING_OPERATOR_UPSTREAM_COMMIT=$(git rev-parse --verify HEAD)
+export CI_CLUSTER_LOGGING_OPERATOR_UPSTREAM_URL=https://github.com/openshift/cluster-logging-operator
+
+#cannot use envsubst here, since replacing images here
```

Comment:
Curious - why can't you use envsubst? The sed script is a lot of duplication and easy to make typo bugs.

Reply:
we dont need these file now